File: php/Scheme/Formula/Interval/IIntervalFormula.php
<?php

namespace Scheme\Formula\Interval;

use Scheme\Interval;
use Scheme\Formula\IFormula;
use Scheme\ListScheme;

interface IIntervalFormula extends IFormula {   
    
    public function getResults(Interval $intv, ListScheme $schemes);
        
    public function setDistance($distance);    
    
    public function getSortingOrder();
    
    public function setSortingOrderAsc();
    
    public function setSortingOrderDesc();
}

File: java/src/formula/interval/IIntervalFormula.java
package formula.interval;

import scheme.Interval;
import formula.IFormula;
import scheme.ListScheme;

public interface IIntervalFormula extends IFormula {   
    
    public double getResults(Interval intv, ListScheme schemes) ;
        
    public void setDistance(int distance);    
    
    public char getSortingOrder();
    
    public void setSortingOrderAsc();
    
    public void setSortingOrderDesc();
}
